src/app/views/communities/view/overview/overview.ts

```diff
@@ -104,11 +105,53 @@ export default class RouteCommunitiesViewOverview extends BaseRouteComponent {
 	emitRefresh() {}
 
 	get routeTitle() {
-		return this.community
-			? this.$gettextInterpolate(`%{ community } Community`, {
-					community: this.community.name,
-			  })
-			: null;
+		if (!this.community) {
+			return null;
+		}
+
+		let title = this.$gettextInterpolate(
+			`%{ name } Community - Fan Art, Videos, Guides, Polls and More`,
```

Comment:
Do it like this `Undertale Community - Fan art, videos, guides, polls, and more`

Reply:
no title case in titles? i mean, sure but any specific reason? seems like common practice